File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/UpdateGrowthPhaseRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class UpdateGrowthPhaseRequest {
    @JsonProperty("GrowthDate")
    public String growthDate;
    @JsonProperty("GrowthPhase")
    public String growthPhase;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("Label")
    public String label;
    @JsonProperty("NewLocation")
    public String newLocation;
    @JsonProperty("NewSublocation")
    public String newSublocation;
    @JsonProperty("NewTag")
    public String newTag;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/UpdateGrowthPhaseRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class UpdateGrowthPhaseRequest(
    @JsonProperty("GrowthDate")
    val growthDate: String? = null,
    @JsonProperty("GrowthPhase")
    val growthPhase: String? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("Label")
    val label: String? = null,
    @JsonProperty("NewLocation")
    val newLocation: String? = null,
    @JsonProperty("NewSublocation")
    val newSublocation: String? = null,
    @JsonProperty("NewTag")
    val newTag: String? = null
)
